auto_update_tests.py

```diff
@@ -222,18 +223,19 @@
         with open(out, 'w') as o: o.write(actual)
         with open(out + '.stdout', 'w') as o: o.write(stdout)
 
-print '\n[ checking binaryen.js testcases... ]\n'
-
-for s in sorted(os.listdir(os.path.join('test', 'binaryen.js'))):
-  if not s.endswith('.js'): continue
-  print s
-  f = open('a.js', 'w')
-  f.write(open(os.path.join('bin', 'binaryen.js')).read())
-  f.write(open(os.path.join('test', 'binaryen.js', s)).read())
-  f.close()
-  cmd = ['mozjs', 'a.js']
-  out = run_command(cmd, stderr=subprocess.STDOUT)
-  open(os.path.join('test', 'binaryen.js', s + '.txt'), 'w').write(out)
+if MOZJS:
+  print '\n[ checking binaryen.js testcases... ]\n'
+
+  for s in sorted(os.listdir(os.path.join('test', 'binaryen.js'))):
+    if not s.endswith('.js'): continue
+    print s
+    f = open('a.js', 'w')
+    f.write(open(os.path.join('bin', 'binaryen.js')).read())
+    f.write(open(os.path.join('test', 'binaryen.js', s)).read())
+    f.close()
+    cmd = ['mozjs', 'a.js']
```

Comment:
OK, but If I make any more changes to this code I'm going to have to go build mozjs just so I can test it =-O